### PR TITLE
Fixed howto div visibility and clinvar links

### PR DESCRIPTION
**Pull request by northwestwitch**
This is a fix to 3 bugs occurring in the clinvar_submissions page

Errors:
- when you click on the howto (instructions to submit) the div is not currently shown
- links to variants are wrong: they are like this /cust/case_id/variant_id and should be like this ```/cust/**case_name**/variant_id'```
- links to cases are wrong: they are like this /cust/case_id and should be like this 
```/cust/**case_name**/```

- [x] code review by dNil
- [x] tests executed by dNil

### CODE REVIEW

File: scout/server/blueprints/cases/templates/cases/clinvar_submissions.html [hunk 1]
```diff
@@ -144,9 +144,9 @@ <h4>Variant data:</h4>
                       <div class="vardata">
                         <div id="vardiv{{subm_variant._id}}" class="panel-body" style="display:none;">
                           {% if subm_variant.category == 'sv' %}
-                          <a href="{{ url_for('variants.sv_variant', institute_id=institute_id, case_name=subm_variant.case_id, variant_id=subm_variant.local_id) }}" target="_blank"><h4>{{subm_variant.chromosome}}_{{subm_variant.breakpoint1}}_{{subm_variant.var_type}}</h4></a>
+                          <a href="{{ url_for('variants.sv_variant', institute_id=institute_id, case_name=subm_obj.cases[subm_variant.case_id], variant_id=subm_variant.local_id) }}" target="_blank"><h4>{{subm_variant.chromosome}}_{{subm_variant.breakpoint1}}_{{subm_variant.var_type}}</h4></a>
```
Comment (dnil) [inline scout/server/blueprints/cases/templates/cases/clinvar_submissions.html]: Looks ok, but to me those hash-ish IDs aren't that informative. How about the display version with chromosome_coord_ref_alt - and truncated if from an SV or larger MNV?

**Comment by dnil** [comment]
..and tests ok: 
Screenshot 2019-06-26 at 09 27 52
Screenshot 2019-06-26 at 10 41 48

File: scout/server/blueprints/cases/templates/cases/clinvar_submissions.html (hunk 1 above)
Comment (northwestwitch) [reply scout/server/blueprints/cases/templates/cases/clinvar_submissions.html]: Good suggestion! It's more informative for the users. I'll change it!

**Comment by northwestwitch** [comment]
Fixed according to suggestion. The page now looks like this:

image

**Comment by northwestwitch** [comment]
Is this OK for merging?

**Comment by dnil** [comment]
:+1:

**Comment by northwestwitch** [comment]
> 👍

Thanks!

**Pull request merged**